db/mainphotos.cql

```diff
@@ -25,7 +25,7 @@ CREATE TABLE mainphotos.property_status (
 DROP TABLE IF EXISTS mainphotos.price_history;
 CREATE TABLE mainphotos.price_history (
   listing_id int,
-  event_date date,
+  event_date timestamp,
```

Comment:
wasn't clear in these commits how you are making use of the `timestamp`. I initially thought you might want to do more granular sorting that `date` cannot achieve but doesn't look like it either. I trust there is some reason unclear to me now.

Reply:
It turns out that date in cassandra doesn't take in time as well so because I was using dates iso formatted, it wouldn't be able to insert if I didn't type the entries as timestamp which takes in both dates and time